src/router/validation/hapiFhirLambdaValidator.ts

```diff
@@ -24,14 +26,18 @@ export default class HapiFhirLambdaValidator implements Validator {
 
     constructor(hapiValidatorLambdaArn: string) {
         this.hapiValidatorLambdaArn = hapiValidatorLambdaArn;
-        this.lambdaClient = new AWS.Lambda();
+        this.lambdaClient = new AWS.Lambda({
+            httpOptions: {
+                timeout: TIMEOUT_MILLISECONDS,
+            },
+        });
     }
 
     async validate(resource: any): Promise<void> {
         const params = {
             FunctionName: this.hapiValidatorLambdaArn,
             InvocationType: 'RequestResponse',
-            Payload: JSON.stringify(resource),
+            Payload: JSON.stringify(JSON.stringify(resource)),
```

Comment:
Do you mean to stringify twice?

Reply:
yes. You always need to stringify at least once since `Payload` MUST be a string(in a sense, Lambda is doing `JSON.parse` once before passing the event to your function handler). 

In our case we want the payload to be the resource serialized as a JSON string, so we end up with 2 calls to `stringify`

Reply:
ahh today I learned -- thanks!